test/services/sync/strategies/TSP/TSP.integration.test.js

```diff
@@ -0,0 +1,75 @@
+const { expect } = require('chai');
+
+const app = require('../../../../../src/app');
+
+const testObjects = require('../../../helpers/testObjects')(app);
```

Comment:
{ cleanup, createTestSchool: createSchool , createTestSystem: createSystem, info: { users: createdTestUsers , accounts: createdAccounts   }  } = 
Why you initialize the included helpers alone?

Reply:
👍